src/components/accessRequests/accessRequestsFilter/accessRequestsFilter.tsx

```diff
@@ -0,0 +1,139 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFilter } from "@fortawesome/free-solid-svg-icons";
+import { Button, Col, Container, Form, Row } from "react-bootstrap";
+
+interface AccessRequestsFilterProps {
+  handleFilter: any;
+  MIN_YEAR: number;
+  MAX_YEAR: number;
+  parseDate: (value: string) => string;
+}
+
+const AccessRequestsFilter = (props: AccessRequestsFilterProps) => {
+  const FORM_GROUP_ROW_CLASS_NAMES = "row mb-3";
+  const LABEL_COL_CLASS_NAMES = "col-4 col-form-label";
+
+  return (
+    <Container className="mb-4 border rounded shadow p-2 mx-0 w-50">
+      <Row>
+        <Col className="fs-1" xs={"auto"}>
+          <FontAwesomeIcon icon={faFilter} />
+        </Col>
+        <Col>
+          <Form
+            onSubmit={(e) => {
+              props.handleFilter();
+              e.preventDefault();
+            }}
+          >
+            <Form.Group className={FORM_GROUP_ROW_CLASS_NAMES}>
+              <Form.Label className={LABEL_COL_CLASS_NAMES}>Dataset</Form.Label>
+              <Col>
+                <Form.Control
+                  type="text"
+                  onChange={(event) => {
+                    props.handleFilter(event.target.value);
+                  }}
+                ></Form.Control>
+              </Col>
+            </Form.Group>
+            <Form.Group className={FORM_GROUP_ROW_CLASS_NAMES}>
+              <Form.Label className={LABEL_COL_CLASS_NAMES}>User</Form.Label>
+              <Col>
+                <Form.Control
+                  type="text"
+                  onChange={(event) => {
+                    props.handleFilter(undefined, event.target.value);
+                  }}
+                ></Form.Control>
+              </Col>
+            </Form.Group>
+            <Form.Group className={FORM_GROUP_ROW_CLASS_NAMES}>
+              <Form.Label className={LABEL_COL_CLASS_NAMES}>
+                Requested date from
+              </Form.Label>
+              <Col>
+                <Form.Control
+                  type="date"
+                  min={props.MIN_YEAR + "-01-01"}
+                  max={new Date().toISOString().split("T")[0]}
+                  onChange={(event) => {
+                    event.target.value = props.parseDate(event.target.value);
+                    props.handleFilter(
+                      undefined,
+                      undefined,
+                      event.target.value
+                    );
+                  }}
+                ></Form.Control>
+              </Col>
+            </Form.Group>
+            <Form.Group className={FORM_GROUP_ROW_CLASS_NAMES}>
+              <Form.Label className={LABEL_COL_CLASS_NAMES}>
+                Requested date until
+              </Form.Label>
+              <Col>
+                <Form.Control
+                  type="date"
```

Comment:
Currently the input controls use DMY format, and the list output is YMD format. Can we do something to make the date format more consistent throughout the app? (Maybe using a date lib like moment?)

Reply:
Not right now. Could be done in another PR. This is getting too long to complete!